pihole

```diff
@@ -269,10 +269,24 @@ piholeCheckoutFunc() {
 }
 
 tricorderFunc() {
+  if [ ! -p "/dev/stdin" ]; then
+    echo "Please do not call Tricorder directly."
+    exit 1
+  fi
+
   if command -v openssl &> /dev/null; then
     openssl s_client -quiet -connect tricorder.pi-hole.net:9998 2> /dev/null < /dev/stdin
   else
-    nc tricorder.pi-hole.net 9999 < /dev/stdin
+    echo "The debug log will be transmitted insecurely via plain-text"
```

Comment:
Is "insecure" really the right wording here? I guess you are talking about "insecure" in the way that others could listen to what you send. I know that with a rogue access point this can be achieved fairly easily.
I guess we should be clear to the user and say something along the lines of "your data will be transported unencrypted and hence there is a possibility that it could be intercepted by a third party".